libraries/octree/src/OctreeConstants.h

```diff
@@ -37,7 +37,7 @@ const int NUMBER_OF_CHILDREN = 8;
 
 const int MAX_TREE_SLICE_BYTES = 26;
 
-const float VIEW_FRUSTUM_FOV_OVERSEND = 0.0f;
+const float VIEW_FRUSTUM_FOV_OVERSEND = 20.0f;
```

Comment:
Note: this 20 degrees is tuned to agree with the 10 degrees of slop in ViewFrustum::isVerySimilar().  Ideally we would adjust the slop there as a function of the value here, but that looked to be be too much work right now: perhaps we'll pass the slop around as parameters in the Glorious Future.